docs/topics/assembly_circuit-breaker-example-application.adoc

```diff
@@ -9,12 +9,16 @@
 [id='example-circuit-breaker-{parameter-runtime-anchor}']
 = {name-example-circuit-breaker} example for {runtime}
 
-include::note-example-unavailable-oso.adoc[leveloffset=+1]
+Example proficiency level: 
+ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
```

Comment:
I believe this is a relic of the reuse in the launcher interface. Please investigate and remove the conditional if it's the case.

Reply:
It appears that this information is still in the booster-catalog (https://github.com/fabric8-launcher/launcher-booster-catalog/blob/master/metadata.yaml#L30). The new Launcher UI does not use it but this information is returned by the backend. So if does not appear in the Launcher UI, can it be removed?

Reply:
Please verify with Engineering if this very file is being used. The metadata file does indeed contain a reference to the proficiency level, but this `ifdef` only modifies whether an `xref` or a `link` is used, which doesn't seem related to me.

Reply:
Confirmed that it is not referenced in any Launcher source files by @quintesse. I'll remove the conditional and the xref and keep the link.